tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -194,6 +194,19 @@ func TestRandomizedSmall(t *testing.T) {
 	th.RunN(ctx, t, numClients, f)
 }
 
+func TestMaintenanceAction(t *testing.T) {
+	ctx := testlogging.ContextWithLevel(t, testlogging.LevelInfo)
+
+	numClients := 1
+	// Run gc action onc
+	runGCFcn := func(ctx context.Context, t *testing.T) { //nolint:thelper
+		t.Log("run maintenance")
+		_, err := eng.ExecAction(ctx, engine.GCActionKey, nil)
+		require.NoError(t, err)
+	}
+	th.RunN(ctx, t, numClients, runGCFcn)
```

Comment:
Does it work? Is maintenance being triggered?

Reply:
Yes. Copied from the log:

`STDERR:
Running full maintenance...
Looking for active contents...
Looking for unreferenced contents...
GC found 0 unused contents (0 B)
GC found 0 unused contents that are too recent to delete (0 B)
GC found 10022 in-use contents (42.2 MB)
GC found 61 in-use system-contents (26.8 KB)
Rewriting contents from short packs...
Total bytes rewritten 873.7 KB
Found safe time to drop indexes: 2024-06-21 11:08:20.307877 -0700 PDT
Dropping contents deleted before 2024-06-21 11:08:20.307877 -0700 PDT
Skipping blob deletion because not enough time has passed yet (59m59s left).
Cleaned up 0 logs.
Cleaning up old index blobs which have already been compacted...
Finished full maintenance.`

Reply:
Is this action being executed by a client?

Reply:
Yes. The test creates a single client, which connects to repo (`kopia repo connect server`), followed by the maintenance action (`kopia maintenance run --full`)

Reply:
I'm confused 😕. That does not seem to match the expected behavior.

Reply:
I see. Can you clarify the expected behavior? I read through the documentation, and was not able to find any reference to a client not being allowed to run maintenance. Is this the expectation you are referring to? Let me know if I am misinterpreted the comment.

Reply:
Here's the explanation from `snapshotter.go`

`// RunGC runs garbage collection on the server repository directly since clients
// are not authorized to do this.
func (mcs *MultiClientSnapshotter) RunGC(ctx context.Context, opts map[string]string) error {
	return mcs.server.RunGC(ctx, opts)
}`

Reply:
What's the alternative to make it clear and explicit?

Reply:
I could replace the function and the call to `RunN` with 
`	th.Engine().TestRepo.RunGC(ctx, nil)
`

This would run the maintenance command . However, it won't be counted in engine action stats.